Tribler/community/tunnel/main.py

```diff
@@ -85,6 +88,8 @@ def clean_messages(self):
             if now - 3600 > self.crawl_message[k]['time']:
                 self.crawl_message.pop(k)
 
+        clean_twisted_observers(globalLogPublisher)
```

Comment:
Oh, I just noticed that you keep passing the type to the function, it's a bit weird, but let's not send yet another PR to dispersy for that.